package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "@fishtank/colors",
-  "version": "3.0.7",
+  "version": "3.0.10",
```

Comment:
Don't bump the version yourself; The version will bump during the deploy process.